MAUI/DateTimePicker/overview.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: About .NET MAUI Date Time Picker control | Syncfusion
+description: Learn here about the overview of Syncfusion .NET MAUI Date Time Picker (SfDateTimePicker) control, its basic features, and its functionalities.
+platform: maui
+control: SfDateTimePicker
+documentation: ug
+---
+
+# Overview of .NET MAUI Date Time Picker (SfDateTimePicker)
+
+Syncfusion .NET MAUI Date Time Picker (SfDateTimePicker) provides a way to pick a date, time or date and time. The Date Time Picker also enables you to set date ranges, date and time formats, and to customize its dialog appearance by configuring its header, footer, and more.
+
+## Key features
+
+* **Header view**: The .NET MAUI Date Time Picker allows the user to add and customize header text.
+
+* **Column header view**: The .NET MAUI Date Time Picker supports a customizable column header view to separate headings for each column.
+
+* **Footer view**: Provides validation buttons (OK and Cancel) in the footer view. The footer text and background color can be customized.
+
+* **Selection view**: Selection view is used to show the selected item and it can be customized.
```

Comment:
* **Selection view**: The selection view is used to show the selected item, and it can be customized.

Reply:
Content modified.